recipes/opencensus/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set version = "0.7.10" %}
+
+package:
+  name: opencensus
+  version: {{ version }}
+
+source:
+  url: https://github.com/census-instrumentation/opencensus-python/archive/v{{ version }}.zip
+  sha256: 4ea0b5bac55c62f9ca5ddccc5acf8160c3575bf58bc5dab7bcb1f5a9cc578e71
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - opencensus-context
+    - google-api-core
+
+test:
+  requires:
+    - pytest
+    - mock
+    - retrying
+    - unittest2
+  source_files:
+    - tests
+  imports:
+    - opencensus
+  commands:
+    - py.test tests/unit -k 'not gets_event and not periodic_task and not multiple_producers and not threaded_export and not producer_error'  # [not win]
```

Comment:
the skipped tests either require a special setup or also fail in the main `opencensus` repository!

Reply:
This commands section isn't needed since there is an import test